kohese-portal/client/src/components/object-editor/format-object-editor/format-object-editor.component.ts

```diff
@@ -52,9 +52,9 @@ export class FormatObjectEditorComponent implements OnInit {
     }
 
     this._object = object;
-    if(this._defferedType) {
-      this.selectedType = this._defferedType;
-      delete this._defferedType;
+    if(this._defferedSelectedType) {
```

Comment:
Since we are in here anyway...can we spell deferred correctly throughout?

Reply:
fixed